Source/ShoutFactory.swift

```diff
@@ -218,15 +220,24 @@ public class ShoutView: UIView {
     announcement.action?()
     silent()
   }
-
+  
+  var subtitleLabelOriginalHeight: CGFloat = 0
```

Comment:
You should probably have this `var` at the top of the file instead of in between method declarations.